codes/c/chapter_dynamic_programming/unbounded_knapsack.c

```diff
@@ -0,0 +1,72 @@
+/**
+ * File: unbounded_knapsack.c
+ * Created Time: 2023-10-02
+ * Author: Zuoxun (845242523@qq.com)
+ */
+
+#include "../utils/common.h"
+
+/* 求最大值 */
+int max(int a, int b) {
+    return a > b ? a : b;
+}
+
+/* 完全背包：动态规划 */
+int unboundedKnapsackDP(int wgt[], int val[], int cap, int wgt_size) {
```

Comment:
`wgtSize`

Reply:
I've changed these naming methods in all my PRs, thx for your review. @Gonglja